src/paulscode/android/mupen64plusae/game/GameLifecycleHandler.java

```diff
@@ -167,7 +176,10 @@ public void onCreateBegin( Bundle savedInstanceState )
         
         // Get app data and user preferences
         mGlobalPrefs = new GlobalPrefs( mActivity );
-        mGamePrefs = new GamePrefs( mActivity, mRomMd5, new RomHeader( mRomPath ) );
+
+        mGamePrefs = new GamePrefs( mActivity, mRomMd5, mRomCrc, mRomHeaderName, RomHeader.countryCodeToSymbol(mRomCountryCode) );
+        mCheatArgs =  mGamePrefs.getCheatArgs();
```

Comment:
I wasn't sure why we passed cheat args around when they could be retrieved right here already.